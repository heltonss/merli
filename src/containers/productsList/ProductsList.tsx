import Product from 'components/Product/Product';
import React, { Component } from 'react';
import Style from './ProductsList.module.scss';

export default class ProductsList extends Component {
  render() {
    const list = [1, 1, 1];
    return (
      <section className={Style.container}>
        {list.map(product => (
          <Product
            key={Math.random()}
            price={'$ 1980'}
            title={'Aple ipode ducks du lroeme fararslfjka '}
            region={'sÃ£o paulo'}
          />
        ))}
      </section>
    );
  }
}
