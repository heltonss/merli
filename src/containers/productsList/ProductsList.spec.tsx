import { shallow } from 'enzyme';
import React from 'react';
import ProductsList from './ProductsList';

describe('ProductsList', () => {
  it('should render without crash', () => {
    const wrapper = shallow(<ProductsList />);
    expect(wrapper).toMatchSnapshot();
  });
});
