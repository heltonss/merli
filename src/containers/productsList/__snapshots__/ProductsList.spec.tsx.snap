// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductsList should render without crash 1`] = `ShallowWrapper {}`;
